gui/sharing/migrations/0011_aio_pthread_remove.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-12-08 04:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+import freenasUI.freeadmin.models.fields
+
+
+def change_cifs_vfsobjects_defaults(apps, schema_editor):
+    cifs_shares = apps.get_model('sharing', 'CIFS_Share').objects.all()
+    if not cifs_shares.exists():
+        return
+
+    for share in cifs_shares:
+        if not share.cifs_vfsobjects:
+            continue
+
+        new_vfs_objects = []
+        new_vfs_objects.extend(
+            filter(lambda v: v not in {'aio_pthread'}, share.cifs_vfsobjects)
+        )
+        share.cifs_vfsobjects = new_vfs_objects
+        share.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sharing', '0007_afp_share_afp_home'),
```

Comment:
should be `0009_afp_share_afp_timemachine_quota_in_gib_instead_of_mib`

Reply:
But why? Its a exact copy of 0008 and it worked, I thought they were idempotent.

Reply:
@william-gr because we don't want do have `0007_afp_share_afp_home` in stable

Reply:
@themylogin I dont get it. What is the problem with that? In stable it references 0009. Is it simply to keep them the same between branches?

Reply:
@themylogin right I but thought you meant the migration code, not the metadata. I will change tomorrow.